script-1.js

```diff
@@ -0,0 +1,165 @@
+// Global variables to track game mode and players' dice rolls
+let gameMode = "roll1"; // other states: 'order1', 'roll2', 'order2', 'result', 'end'
```

Comment:
As a general point for all your global variables, they should all be `const` unless you specifically reassign them to another value later on in your code